### DISCUSSION TITLE
Node statisics or Queue information not available in the cluster node

### DISCUSSION THREAD

**Discussion by anmol1vw13** [rabbitmq/rabbitmq-server#4270] [Other]
I have a 5 node rabbitmq cluster running on rabbitmq version 3.8.5, erlang version 21.3.
I add a 6th node into the cluster running on rabbitmq version 3.8.27 erlang version 24.2.2
There's an issue w.r.t the management UI.
**Management UI of the existing cluster**
<img width="1460" alt="Screenshot 2022-03-10 at 5 48 21 PM" src="https://user-images.githubusercontent.com/13599208/158109686-c6153b6f-5d93-4a9b-9d1c-f9f3b882a2fb.png">
Node statistics of the 6th node rabbit@g-lambda-lambda-rabbitmq-rabbitmq-a-2002 aren't available.
**Management UI of the new node**
<img width="1463" alt="Screenshot 2022-03-10 at 5 53 57 PM" src="https://user-images.githubusercontent.com/13599208/158109754-9dab1b94-ee31-4ef1-9775-1c94d73c6e5c.png">
Node statistics of only the 6th node i.e. rabbit@g-lambda-lambda-rabbitmq-rabbitmq-a-2002 are available.

Also, a similar issue is seen with the queues page as well. The information for the queues for which the 6th node (rabbit@g-lambda-lambda-rabbitmq-rabbitmq-a-2002) is the leader, shows up as NaN in the management UI of the existing cluster.

<img width="1536" alt="Screenshot 2022-03-10 at 5 55 31 PM" src="https://user-images.githubusercontent.com/13599208/158109833-f4e013cf-fa81-4078-be70-5906e602207c.png">
While the management UI of the 6th node only shows the information for the queues for which it is the leader and NaN for the rest.
<img width="1509" alt="Screenshot 2022-03-10 at 5 58 48 PM" src="https://user-images.githubusercontent.com/13599208/158109852-513b02ec-8fac-46c8-b1e3-8be988842117.png">
A similar issue was also posted by someone else on StackOverflow in 2021 but didn't find a concrete answer to it.
https://stackoverflow.com/questions/69881325/rabbitmq-node-statistics-not-available-in-cluster-node.

I had posted the same question on the rabbitmq users google group but haven't received a response yet.
https://groups.google.com/g/rabbitmq-users/c/GnMCIpVTE9U

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
Please upgrade all nodes to a recent patch version and see node logs. `3.8.24` management plugin may or may not be compatible with every single patch release from the last couple of years (`3.8.5` is from June 2020). Nodes retrieve stats from their peers, after all.

For some metrics, a relevant [feature flag](https://rabbitmq.com/feature-flags.html) must be enabled. Nodes from two years may or may not support it. But all evidence is that stats data stores or inter-node stats query API in those two version may be subtly different.
